docs/platform-engineers/workflow/working-mechanism.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Working Mechanism
+---
+
+This document will give a brief introduction to the core mechanisms of KubeVela Workflow.
+
+## Running Mode
+
+The execution of workflow has two different running modes: **DAG** mode and **StepByStep** mode. In DAG mode, all steps in the workflow will execute concurrently. They will form a dependency graph for running according to the Input/Output in the step configuration automatically. If one workflow step has not met all its dependencies, it will wait for the conditions. In StepByStep mode, all steps will be executed in order. In KubeVela v1.2+, the defaut running mode is StepByStep.
+
```

Comment:
how to specify the running mod?

Reply:
We do not allow manually specifying running mode now.

Reply:
yes, I mean it's not clear in the doc

Reply:
OK.